students/20th/wonyoungcho/westagram/js/main.js

```diff
@@ -0,0 +1,29 @@
+'user strict';
+
+const idName = 'wychrischo ';
+const comment = document.querySelector('.postUserContainer');
+const addButton = document.querySelector('.commentsPostButton');
+const textArea = document.querySelector('.commentsInputComment');
+
+function commentAdd () {
+  const newComment = textArea.value;
+  const newDiv = document.createElement('div');
+  newDiv.className = 'post-user';
+
+  comment.append(newDiv);
+  newDiv.innerHTML = `<div class = 'postUser'><span class = 'postUserId'>${idName}</span> ${newComment}</div>`;
```

Comment:
```suggestion
  newDiv.innerHTML = `
    <div class='postUser'>
      <span class='postUserId'>${idName}</span>${newComment}
    </div>`;
```
- 줄바꿈 해주면 더 읽기 좋은 코드가 될 것 같습니다.